storybook/pages/ModelEntryPage.qml

```diff
@@ -20,6 +22,32 @@ Control {
         sourceModel: usersModel
         key: "pubKey"
         value: pubKeySelector.currentText
+        
+        onItemChanged: signalsModel.append({        signal: "Item changed",     value: "", row: itemData.row, item: itemData.item, roles: JSON.stringify(itemData.roles), available: itemData.available})
+        onRowChanged: signalsModel.append({         signal: "Row changed",      value: "", row: itemData.row, item: itemData.item, roles: JSON.stringify(itemData.roles), available: itemData.available})
+        onAvailableChanged: signalsModel.append({   signal: "Available changed",value: "", row: itemData.row, item: itemData.item, roles: JSON.stringify(itemData.roles), available: itemData.available})
+        onRolesChanged: signalsModel.append({       signal: "Roles changed",    value: "", row: itemData.row, item: itemData.item, roles: JSON.stringify(itemData.roles), available: itemData.available})
+    }
+
+    Instantiator {
+        model: itemData.roles
+        delegate: QtObject {
+            property var connection: {
+                return Qt.createQmlObject(`
+                    import QtQml 2.15
+                    Connections {
```

Comment:
I was trying with `itemData.item[modelData + "Changed"].connect` but for some reason it doesn't work with QQmlPropertyMap... :thinking:

Reply:
ohh, IDK. Didn't think of this possibility..Not sure why it wouldn't work

Reply:
nvm, your solution works nicely :)